index.js

```diff
@@ -59,14 +59,47 @@ animalToRemove.remove()
 /************************** EVENTS JS MINI CHALLENGE ******************************/
 
 /***** Deliverable 1 *****/
+
 function toggleColor(element) {
-    if (element.style.color === "green") {
+    if (element.style.color === "red") {
         element.style.color = "black"
     } else {
-        element.style.color = "green"
+        element.style.color = "red"
     }
 }
 
+header.addEventListener('click', function() {
+    toggleColor(header)
+})
+
 /***** Deliverable 2 *****/
+const likeButton = document.querySelector('button.like-button')
+
+likeButton.addEventListener("click", function() {
+    likes.textContent = `${traveler.likes++} Likes`
+})
```

Comment:
Great work! I would create a separate callback function to pass in to the event listener. Even though it's not a ton of logic, it will prepare you for the next steps when you will be submitting data to the backend. You will want to take the current likes off the element on the page, turn that into an integer, then increase by 1.

My steps are as follows:

1. First find the like button element on the page

2. Then add an event listener to the like button

3. We are listening for a `click` event, and we will pass a callback function to be invoked when the event fires.

4. Then we create the function `addNewLike` which will handle all the functionality.

The `addNewLike` function has a few steps:

1. find the `p` tag where the current number of likes is stored on the page
2. extract the number out of the string, make sure to convert it to a number with `parseInt()`
3. add 1 to that number.
4. Create a new string with the new number
5. update the `p` tag to have the newly created string as its `innerText`.

```
const likeBtn = document.querySelector(".like-button");

function addNewLike(e){
    const likesPTag = e.target.previousElementSibling;
    const likesText = likesPTag.innerText;
    let currentLikes = parseInt(likesText.split(" ")[0]);
    currentLikes += 1;
    const newLikesString = `${currentLikes} Likes`;
    likesPTag.innerText = newLikesString;
}

likeBtn.addEventListener("click", addNewLike)
```

With ES6 notation

```
const likeBtn = document.querySelector(".like-button");

const addNewLike = e => {
    const likesPTag = e.target.previousElementSibling;
    let currentLikes = parseInt(likesPTag.innerText.split(" ")[0]);
    currentLikes += 1;
    const newLikesString = `${currentLikes} Likes`;
    likesPTag.innerText = newLikesString;
}

likeBtn.addEventListener("click", addNewLike)

```